js/apps/system/_admin/aardvark/APP/react/src/views/views/forms/inputs/LinkPropertiesInput.tsx

```diff
@@ -31,140 +36,123 @@ const LinkPropertiesInput = ({
     });
   };
 
-
-  const storeIdValues = get(formState, 'storeValues') === 'id';
+  const storeIdValues = get(formState, "storeValues") === "id";
   const hideInBackgroundField = disabled || basePath.includes(".fields");
 
   return (
-    <Grid>
-      <Cell size={"1-2"}>
-        <Grid>
-          <Cell size={"1"} style={{ marginBottom: 24 }}>
-            <span
-              style={{
-                float: "left",
-                width: "84%"
-              }}
-            >
-              <label>Analyzers</label>
-              <AnalyzersDropdown
-                basePath={basePath}
-                isDisabled={!!disabled}
-                formState={formState}
-              />
-            </span>
-          </Cell>
+    <Grid
+      templateColumns="1fr 1fr"
+      columnGap="10"
+      border="2px solid"
+      borderColor="gray.300"
+      paddingX="4"
+      paddingY="2"
+      background="gray.200"
+    >
+      <Stack spacing="4">
+        <Stack>
+          <FormLabel margin="0">Analyzers</FormLabel>
+          <AnalyzersDropdown
+            basePath={basePath}
+            isDisabled={!!disabled}
+            formState={formState}
+          />
+        </Stack>
 
-          <Cell size={"1"}>
-            <span
-              style={{
-                float: "left",
-                width: "84%"
-              }}
-            >
-              <label>Fields</label>
-              <FieldsDropdown
-                basePath={basePath}
-                isDisabled={!!disabled}
-                formState={formState}
+        <Stack>
+          <HStack alignItems="center">
+            <FormLabel margin="0" position="relative" top="1">
+              Fields
+            </FormLabel>
+            <InfoTooltip label="Add field names that you want to be indexed here. Click a field name to set field details" />
```

Comment:
Does this make sense contextually ?
```suggestion
            <InfoTooltip label="Add field names that you want to be indexed here. Click  on a field name to set field details" />
```